pkg/archive/filter.go

```diff
@@ -46,7 +46,14 @@ func tryProcFilter(args []string, input io.Reader, cleanup func()) (io.ReadClose
 	go func() {
```

Comment:
This is a bit of a tangent but I am not sure why we're not using 
https://pkg.go.dev/os/exec#Cmd.StdinPipe above?

Reply:
I've not used `StdoutPipe` to report explicitly the error with `CloseWithError`.  I'll need to check if that is the same behavior with `StdoutPipe`